edgexfoundry/device-usb-camera
# refactor!: Replace internal topics from config with new constants

BREAKING CHANGE: Internal topics no longer configurable, except the base topic.







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
1. run edgex non-secure stack
3. build device service in docker- `make docker`
4. Add the device service to the compose file `make gen <previous options> ds-usb-camera`
5. Run `make up` to start the device service in docker
6. Verify the log has on following start-up :
```
msg="Subscribing to command requests on topic: edgex/device/command/request/device-usb-camera/#"
msg="Responses to command requests will be published on topic: edgex/response/device-usb-camera/<requestId>"
"Subscribing to System Events on topic: edgex/system-events/core-metadata/+/+/device-usb-camera/#
```


## New Dependency Instructions (If applicable)
